include/arch/arm/armv/armv8-a/64/armv/machine.h

```diff
@@ -16,6 +16,11 @@ static inline void dsb(void)
     asm volatile("dsb sy" ::: "memory");
 }
 
+static inline void dsb_ishst(void)
+{
+    asm volatile("dsb ishst" ::: "memory");
+}
+
```

Comment:
An implementation will also need to be added for armv7 to avoid compilation errors I expect. I'll just switch on the `hw_builds` flag for this PR to build all configurations

Reply:
Do I need to add it for armv7-a as well? Let me know ;)

Reply:
> Do I need to add it for armv7-a as well? Let me know ;)

Yes. See the build errors the Github CI give on ARMv7 at https://github.com/seL4/seL4/actions/runs/4766337058/jobs/8473219222?pr=1014
```
...
SABRE_release_SMP_gcc_32
...
[160/323] Building C object kernel/CMakeFiles/kernel.elf.dir/kernel_all.c.obj
...
  /github/workspace/kernel/include/arch/arm/arch/machine/hardware.h:29:6: error: implicit declaration of function ‘dsb_ishst’ [-Werror=implicit-function-declaration]
     29 |      dsb_ishst(); \
        |      ^~~~~~~~~
```